config/software/postgresql.rb

```diff
@@ -15,7 +15,7 @@
 #
 
 name "postgresql"
-default_version "9.2.10"
+default_version "9.6.24"
```

Comment:
[Optional] The latest version of `omnibus-software` still uses "9.6.22". Does upgrading all the way to "9.6.24" buy us anything? If not, can we stay in sync?

Reply:
This should be fine. There is another post that ran into the same openssl problem that ends up using this version. We are not alone.